lib/ReactTags.js

```diff
@@ -127,7 +127,7 @@ class ReactTags extends Component {
     let { query, selectedIndex, suggestions } = this.state;
 
     // hide suggestions menu on escape
-    if (e.keyCode === Keys.ESCAPE) {
+    if (e.key === "Escape") {
```

Comment:
@AndrewFinlay can you update the constant `Keys` declared in this file with these values instead of using hard coded values in this file?

Reply:
Will do

Reply:
Looking at it more closely, I think that this breaks props.delimiters.